### PR TITLE
bugfix: Process of bluetooth mapper scheduler seems wrong 

### LINKED ISSUE #1824: Process of bluetooth mapper scheduler seems wrong
code: kubeedge/mapper/bluetooth_mapper/scheduler/scheduler.go: line 81
issue 1:
actionExists will always be true once it enters  line 85; And it will break once the first action in schedule.Actions was deleted(line 83 mismatch).

issue 2:
No process when the scheduler was deleted.
bug1

### PR DESCRIPTION
**Pull request by sailorvii**
The action in one schedule can be more than one. "actionExists" will always be true once it enters line 85,
which is not expected. Morever, if one action cannot be found, other actions should be run.

**What type of PR is this?**
 /kind bug

**What this PR does / why we need it**:
The process is wrong.

**Which issue(s) this PR fixes**:
Fixes #1824 

**Special notes for your reviewer**:

**Does this PR introduce a user-facing change?**:
None

### CODE REVIEW

**Comment by kadisi** [comment]
looks good.

cc @fisherxu @kevin-wangzefeng

File: mappers/bluetooth_mapper/scheduler/scheduler.go [hunk 1]
```diff
@@ -95,8 +95,8 @@ func (schedule *Schedule) performScheduleOperation(actionManager []actionmanager
 			schedule.Interval = defaultEventFrequency
 		}
 		if !actionExists {
-			klog.Errorf("Action %s does not exist. Exiting from schedule !!!", actionName)
-			break
+			klog.Errorf("Action %s does not exist.", actionName)
+			continue
```
Comment (fisherxu) [inline mappers/bluetooth_mapper/scheduler/scheduler.go:99]: Thanks for the fixing, and what's the usage of `continue` here? No matter the actionExists  is true or false, the for will continue

Comment (sailorvii) [reply mappers/bluetooth_mapper/scheduler/scheduler.go:99]: Yes, for there's a sleep after each schedule. When there's no action, I thought the sleep is not needed.
BTW, move this schedule.Interval setting out of this for rotation.

**Pull request merged**